test/accelerators/NNPA/numerical/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(ONNX_NUMERICALTEST_SRC_DIR ${ONNX_MLIR_SRC_ROOT}/test/numerical)
+set(ONNX_NUMERICALTEST_BIN_DIR ${ONNX_MLIR_BIN_ROOT}/test/numerical)
+
+add_custom_target(numerical-nnpa)
+set_target_properties(numerical-nnpa PROPERTIES FOLDER "Tests")
+
+set(ZDNN_LIB "${ONNX_MLIR_BIN_ROOT}/src/Accelerators/NNPA/zDNN/src/zdnn/zdnn/lib/libzdnn.so")
+
+add_custom_target(check-onnx-numerical-nnpa
+  COMMENT "Running the ONNX-MLIR numerical regression tests for NNPA"
+  COMMAND "${CMAKE_CTEST_COMMAND}" -L numerical-nnpa --output-on-failure -C $<CONFIG> --force-new-ctest-process
+  USES_TERMINAL
+  DEPENDS numerical-nnpa
+  )
+set_target_properties(check-onnx-numerical-nnpa PROPERTIES FOLDER "Tests")
+# Exclude the target from the default VS build
+set_target_properties(check-onnx-numerical-nnpa PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD ON)
```

Comment:
Is it for Windows? I am not sure, just looking at the word `VS`.

Reply:
I removed the line. Thanks!